### PR TITLE
remove BZ 2181546 from pao e2e skip list

### PR DESCRIPTION
**Pull request by yanirq**
resolved in OCPBUGS-10970

### CODE REVIEW

File: test/e2e/performanceprofile/functests/1_performance/irqbalance.go [hunk 1]
```diff
@@ -165,7 +165,6 @@ var _ = Describe("[performance] Checking IRQBalance settings", Ordered, func() {
 				Skip("this test needs dynamic IRQ balancing")
 			}
 
-			testutils.KnownIssueBugzilla(2181546)
 			testutils.KnownIssueJira("OCPNODE-1538")
```
Comment (Tal-or) [inline test/e2e/performanceprofile/functests/1_performance/irqbalance.go:168]: What about?

Comment (yanirq) [reply test/e2e/performanceprofile/functests/1_performance/irqbalance.go:168]: this is a different bug

**Comment by yanirq** [comment]
test lanes not affected from this change
/override ci/prow/e2e-aws-ovn 
/override ci/prow/e2e-gcp-pao 
/override ci/prow/e2e-gcp-pao-updating-profile
/override ci/prow/e2e-upgrad

**Comment by yanirq** [comment]
test lanes not affected from this change
/override ci/prow/e2e-aws-ovn
/override ci/prow/e2e-gcp-pao
/override ci/prow/e2e-gcp-pao-updating-profile
/override ci/prow/e2e-upgrade

**Pull request merged**